ci/tests.yaml

```diff
@@ -19,3 +19,4 @@ dependencies:
   - setuptools_scm
   - pip:
     - pyradiosky
+    - lunarsky
```

Comment:
we should probably add this to the `environment.yaml` file in the top level of the repo as well. And we should add it to the extras_require options `all` and `dev` and consider making a new option `moon`.

Reply:
Thanks for the suggestion! I've added it to the environment.yaml, and the extras_require options, and added a `moon` option.